lib/hawkular/operations/operations_api.rb

```diff
@@ -105,7 +127,8 @@ def connect
         client.on(:message, once: true) do |msg|
           parsed_message = msg.data.to_msg_hash
 
-          logger.log("Sent WebSocket message: #{parsed_message}")
+          # Not working with websocket-vcr
```

Comment:
I guess it can be done [like in the handle_message method](https://github.com/israel-hdez/hawkular-client-ruby/blob/48aeda8b1beacf91b7bd2fea64968403fd0b1755/lib/hawkular/operations/operations_api.rb#L399) where a new instance of logger is created.

BTW, I just realized that [the `@session_id`](https://github.com/israel-hdez/hawkular-client-ruby/blob/48aeda8b1beacf91b7bd2fea64968403fd0b1755/lib/hawkular/operations/operations_api.rb#L135) is not being set because is also out of scope. It's not used anywhere in the gem. So, not sure if we should worry about it. If we want to fix it, the better way should be to place a PR on event_emitter hoping that the author is willing to change that.